scm/src/scm_input.F90

```diff
@@ -1630,7 +1630,7 @@ subroutine get_case_init_DEPHY(scm_state, scm_input)
      call NetCDF_read_var(ncid, "wprvp_s",    .False., input_force_wprvp)
      call NetCDF_read_var(ncid, "wprtp_s",    .False., input_force_wprtp)
   else if (trim(input_surfaceForcingMoist) == 'surface_flux') then
-     call NetCDF_read_var(ncid, "hfls",       .False., input_force_sfc_lat_flx)
+     call NetCDF_read_var(ncid, "hfls",       .False., input_force_sfc_sens_flx)
```

Comment:
@bluefinweiwei What is the reason for this change? `hfls` is supposed to be the surface latent heat flux, not sensible.

Reply:
@grantfirl This change should be discarded. About this line, yes, you're right that it should be "lat" not "sens". Also, any change for this line is actually not necessary for COMBLE, since its surface forcing is surface temperature "ts" not heat fluxes. I am not sure why this was not updated and pushed, which might be a mistake by me. Sorry for the confusion!